### DISCUSSION TITLE
Capacity/generation output for different cooling technology

### DISCUSSION THREAD

**Discussion by cassiemavs** [NREL/ReEDS-2.0#157] [Q&A: Features] [answered]
Is there any output for generation/capacity sorted by cooling technology? I do see capacity by region and fuel technology. Or if there is any model setting that needs to be switched on?

**Comment by wesleyjcole** [comment]
The cooling technologies are in the ctt set, and the water source types are in the wst set.  When cooling technologies are turned on, the i set (the technology set) will include these other elements.  You can see that in the plant database (inputs/capacitydata/ReEDS_generator_database_final_EIA-NEMS.csv) in the column coolingwatertech.

**Reply by cassiemavs** [reply]
Is "GSw_CoolingTechMults = 1" in the cases.csv for switching on the cooling technologies? Or other paramters that need to be changed?

**Reply by wesleyjcole** [reply, marked as answer]
There are a variety of ways to represent cooling water.  You can turn on only the cooling water representation to get an accounting of water use, or turn on the associated constraints, which might impact the generation mix.  There are also switches associated with water needs for PSH.  The primary switches that I've used are GSw_WaterMain and GSw_WaterUse.

A few notes with water:
- The model grows significantly in size when water constraints are turned on.
- For the 2022 version of the model, upgrades are not compatible with the water constraints, so GSw_Upgrades needs to be set to 0.
